zerver/migrations/0327_merge_20210603_1506.py

```diff
@@ -0,0 +1,15 @@
+# Generated by Django 3.2.2 on 2021-06-03 15:06
+
+from typing import Any, List
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zerver", "0325_realm_jitsi_server_url"),
+        ("zerver", "0326_alter_realm_authentication_methods"),
+    ]
+
+    operations: List[Any] = []
```

Comment:
These migrations aren't renumbered properly; see https://zulip.readthedocs.io/en/latest/subsystems/schema-migrations.html.

(One shouldn't have these merge migrations).

Reply:
Alright, will do it together with other possible changes within this commit.